src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  status: [],
+};
+
+const categoriesSlice = createSlice({
+  name: 'status',
+  initialState,
+  reducers: {
+    checkStatus: (state) => {
+      // eslint-disable-next-line no-param-reassign
```

Comment:
- [ ] [Optional] Linters are to make our codebase cleaner instead of disabling it try fixing the error :+1: